chemicaltoolbox/fpocket/fpocket.xml

```diff
@@ -0,0 +1,140 @@
+<tool id="fpocket" name="fpocket" version="0.1.0">
+    <description>- find potential binding sites in protein structures</description>
+    <requirements>
+        <requirement type="package" version="3.1.3">fpocket</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+
+        ln -s '$input' ./input.pdb &&
+        fpocket
+            -f ./input.pdb
+            #if $inp.pocket_type == 'channel':
```

Comment:
just checking, `small_mol` is not needed here?

Reply:
this could be a elif ...

Reply:
no, small_mol is the default

Reply:
will update to elif